libPhoneNumber/Internal/NBPhoneNumberUtil+Category.h

```diff
@@ -0,0 +1,23 @@
+//
+//  NBPhoneNumberUtil+Category.h
+//  libPhoneNumber
+//
+//  Created by Paween Itthipalkul on 12/1/17.
+//  Copyright © 2017 Google LLC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "NBPhoneNumberUtil.h"
+
+@class NBMetadataHelper, NBRegExMatcher;
+
+/**
+ This interface exposes properties used in NBPhoneNumberUtil categories.
+ */
+@interface NBPhoneNumberUtil()
```

Comment:
normally we don't expose a class extension in a header.

Reply:
Removed the file